ThunderCloud/ContentController.swift

```diff
@@ -366,37 +366,39 @@ public class ContentController: NSObject {
     
     /// Downloads a full storm content bundle, this will clear all directories and will also mark the downloaded bundle as the 'initial' bundle timestamp
     /// so that we can avoid downloading any post-landmark publishes from content-available notifications!
-    /// - Parameter progressHandler: A closure which will be called with the progress of the bundle download
-    public func downloadFullBundle(with progressHandler: ContentUpdateProgressHandler?) {
+    /// - Parameter buildTimestamp: The timestamp of the build, used to make sure we don't bypass any landmark publishes
```

Comment:
Might be worth documenting what these seconds represent, unix time?

Reply:
Yep, good shout!